cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -567,6 +583,10 @@ private class CampaignManagerImpl(
             dataStorage.ccpaConsentUuid = value
         }
 
+    override var usNatUuid: String?
+        get() = usNatConsentStatus?.uuid
+        set(value) { usNatConsentStatus?.uuid = value }
```

Comment:
I'm not sure if that way you will achieve what you want to, I don't think this line will actually update the `uuid` of the `usNatConsentStatus` from the `DataStorage`. Meaning that, if we set `usNatUuid` and then call `usNatConsentStatus?.uuid` we will get different results.

I think it should be something like:
```
set(value) {
    usNatConsentStatus = usNatConsentStatus.copy(uuid = value)
}
```

Reply:
changing like this
`
override val usNatUuid: String?
        get() = usNatConsentStatus?.uuid
`

Reply:
I'll just delete this property